google/cloud/speech/client.py

```diff
@@ -19,30 +19,8 @@
 from google.cloud._helpers import _to_bytes
 from google.cloud import client as client_module
 from google.cloud.speech.connection import Connection
-
-
-class Encoding(object):
-    """Audio encoding types.
-
-    See:
-    https://cloud.google.com/speech/reference/rest/v1beta1/\
-    RecognitionConfig#AudioEncoding
-    """
-
-    LINEAR16 = 'LINEAR16'
-    """LINEAR16 encoding type."""
-
-    FLAC = 'FLAC'
-    """FLAC encoding type."""
-
-    MULAW = 'MULAW'
-    """MULAW encoding type."""
-
-    AMR = 'AMR'
-    """AMR encoding type."""
-
-    AMR_WB = 'AMR_WB'
-    """AMR_WB encoding type."""
+from google.cloud.speech.encoding import Encoding
+from google.cloud.speech.operation import Operation
```

Comment:
NOTE: You could've done this refactor in a standalone PR and made this PR easier to review.